hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncNonMetaRegionLocator.java

```diff
@@ -50,6 +50,7 @@
 import java.util.concurrent.TimeUnit;
 import org.apache.commons.lang3.ObjectUtils;
 import org.apache.hadoop.hbase.CatalogFamilyFormat;
+import org.apache.hadoop.hbase.CatalogReplicaMode;
```

Comment:
This is because we move CatalogReplicaMode to another package?

Reply:
Yes, since it is moved to another package, it needs to be imported.